Setup.hs

```diff
@@ -1,11 +1,15 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE TupleSections #-}
 
+#ifndef MIN_VERSION_Cabal
```

Comment:
Thank you. Am I right that "Custom-setup" stanza is only supported in cabal-install >= 1.24 ?